package.json

```diff
@@ -20,13 +21,16 @@
   "dependencies": {
     "bcryptjs": "^2.4.3",
     "cors": "^2.8.5",
+    "cross-env": "^6.0.0",
     "dotenv": "^8.1.0",
     "express": "^4.17.1",
     "helmet": "^3.21.0",
+    "jest": "^24.9.0",
     "jsonwebtoken": "^8.5.1",
     "knex": "^0.19.4",
     "morgan": "^1.9.1",
     "nodemon": "^1.19.2",
-    "sqlite3": "^4.1.0"
+    "sqlite3": "^4.1.0",
+    "supertest": "^4.0.2"
   }
```

Comment:
You'll probably have to install postgres as well so that your database doesn't reset in heroku